src/sgame/sg_bot_util.cpp

```diff
@@ -365,7 +365,7 @@ float BotGetResupplyScore( gentity_t *self )
 			ASSERT_UNREACHABLE();
 	}
 	float timeDist = dist / GetMaximalSpeed( self );
-	return ( 1 + 5 * self->botMind->botSkill.level ) * ( 1 - percentAmmo ) / sqrt( timeDist );
+	return ( 1 + 5 * self->botMind->botSkill.level ) * ( needPoison ? 1 : ( 1 - percentAmmo ) ) / sqrt( timeDist );
```

Comment:
The poison thing could be kind of broken for large maps; you may not be able to get to the human base before the poison runs out.

Reply:
Good point.
But this is easy to counter in the BT itself by checking distance to BOOSTER before checking for the score. I believe this is something that should be done more often anyway, notably for the actions `heal` and `equip`.

IIRC the logic here is the same as in `heal` for the distance (or was when I initially implemented this).